test/controllers/users_logix_controller_test.rb

```diff
@@ -0,0 +1,48 @@
+require 'test_helper'
+
+class UsersLogixControllerTest < ActionDispatch::IntegrationTest
```

Comment:
✂️

Reply:
removed this file as its no longer required